src/latex/agda-latex-macros.sty

```diff
@@ -336,38 +336,9 @@
-\newcommand{\LedgerPrelude}{\href{https://github.com/IntersectMBO/formal-ledger-specifications/blob/master/src/Ledger/Prelude.lagda}{\AgdaModule{Ledger.Prelude}}\xspace}
-\newcommand{\LedgerProperties}{\href{https://github.com/IntersectMBO/formal-ledger-specifications/blob/master/src/Ledger/Properties.lagda}{\AgdaModule{Ledger.Properties}}\xspace}
-\newcommand{\LedgerRatify}{\href{https://github.com/IntersectMBO/formal-ledger-specifications/blob/master/src/Ledger/Ratify.lagda}{\AgdaModule{Ledger.Ratify}}\xspace}
-\newcommand{\LedgerRatifyProperties}{\href{https://github.com/IntersectMBO/formal-ledger-specifications/blob/master/src/Ledger/Ratify/Properties.lagda}{\AgdaModule{Ledger.Ratify.Properties}}\xspace}
-\newcommand{\LedgerScript}{\href{https://github.com/IntersectMBO/formal-ledger-specifications/blob/master/src/Ledger/Script.lagda}{\AgdaModule{Ledger.Script}}\xspace}
-\newcommand{\LedgerScriptValidation}{\href{https://github.com/IntersectMBO/formal-ledger-specifications/blob/master/src/Ledger/ScriptValidation.lagda}{\AgdaModule{Ledger.ScriptValidation}}\xspace}
-\newcommand{\LedgerTokenAlgebra}{\href{https://github.com/IntersectMBO/formal-ledger-specifications/blob/master/src/Ledger/TokenAlgebra.lagda}{\AgdaModule{Ledger.TokenAlgebra}}\xspace}
-\newcommand{\LedgerTokenAlgebraValueSet}{\href{https://github.com/IntersectMBO/formal-ledger-specifications/blob/master/src/Ledger/TokenAlgebra/ValueSet.lagda}{\AgdaModule{Ledger.TokenAlgebra.ValueSet}}\xspace}
-\newcommand{\LedgerTransaction}{\href{https://github.com/IntersectMBO/formal-ledger-specifications/blob/master/src/Ledger/Transaction.lagda}{\AgdaModule{Ledger.Transaction}}\xspace}
-\newcommand{\LedgerTypesEpoch}{\href{https://github.com/IntersectMBO/formal-ledger-specifications/blob/master/src/Ledger/Types/Epoch.lagda}{\AgdaModule{Ledger.Types.Epoch}}\xspace}
-\newcommand{\LedgerTypesGovStructure}{\href{https://github.com/IntersectMBO/formal-ledger-specifications/blob/master/src/Ledger/Types/GovStructure.lagda}{\AgdaModule{Ledger.Types.GovStructure}}\xspace}
-\newcommand{\LedgerUtxo}{\href{https://github.com/IntersectMBO/formal-ledger-specifications/blob/master/src/Ledger/Utxo.lagda}{\AgdaModule{Ledger.Utxo}}\xspace}
-\newcommand{\LedgerUtxoProperties}{\href{https://github.com/IntersectMBO/formal-ledger-specifications/blob/master/src/Ledger/Utxo/Properties.lagda}{\AgdaModule{Ledger.Utxo.Properties}}\xspace}
-\newcommand{\LedgerUtxow}{\href{https://github.com/IntersectMBO/formal-ledger-specifications/blob/master/src/Ledger/Utxow.lagda}{\AgdaModule{Ledger.Utxow}}\xspace}
-\newcommand{\LedgerUtxowProperties}{\href{https://github.com/IntersectMBO/formal-ledger-specifications/blob/master/src/Ledger/Utxow/Properties.lagda}{\AgdaModule{Ledger.Utxow.Properties}}\xspace}
+\NewDocumentCommand\LedgerModule{m}{%
+  \href{https://github.com/IntersectMBO/formal-ledger-specifications/blob/master/src/Ledger/#1.lagda}{\AgdaModule{Ledger.#1}}%
+}
```

Comment:
Nice!  This is exactly what I had in mind, but wasn't working when I tried to define it with `\newcommand`.  Interesting it works with `\NewDocumentCommand`. 🤔

Note that it's not general enough to handle, e.g.,

```
\newcommand{\LedgerChainProperties}{\href{https://github.com/IntersectMBO/formal-ledger-specifications/blob/master/src/Ledger/Chain/Properties.lagda}{\AgdaModule{Ledger.Chain.Properties}}\xspace}
```

...but that's okay.  We can leave that one as is for now.